util/audit-packages.sh

```diff
@@ -1,15 +1,5 @@
 #!/bin/bash
 
-source $(dirname $0)/packages.env
 pushd $(dirname $0)/..
```

Comment:
Possible to remove this script and replace with `npm audit` direct usage now

Reply:
As above